edq/core/config.py

```diff
@@ -37,6 +39,18 @@ def __eq__(self, other: object) -> bool:
     def __str__(self) -> str:
         return f"({self.label}, {self.path})"
 
+def write_config_to_file(file_path: str, config_to_write: typing.Dict[str, str]) -> None:
```

Comment:
Why "file_path" instead of "path"?
We normally just use "path" (unless we need more context).